src/test/java/core/basesyntax/servise/WriterServiceImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.servise;
+
+import core.basesyntax.files.Reader;
+import core.basesyntax.files.ReaderFileImpl;
+import core.basesyntax.files.Writer;
+import core.basesyntax.files.WriterFileImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterServiceImplTest {
+    private static final String TEST_FILE = "src/main/resources/test.csv";
+    private static final String TITLE = "fruit,quantity";
+    private static final String LINE_01 = "banana,152";
```

Comment:
The same as above

Reply:
But this data isn't source data. It is needed to create our expected result